src/bin/cargo-miri.rs

```diff
@@ -419,25 +424,26 @@ fn inside_cargo_rustc() {
     };
     args.splice(0..0, miri::miri_default_args().iter().map(ToString::to_string));
 
-    // See if we can find the cargo-miri markers.  Those only get added to the binary we want to
-    // run.  They also serve to mark the user-defined arguments, which we have to move all the way to the
-    // end (they get added somewhere in the middle).
-    let needs_miri = if let Some(begin) = args.iter().position(|arg| arg == "cargo-miri-marker-begin") {
-        let end = args.iter().position(|arg| arg == "cargo-miri-marker-end").expect("Cannot find end marker");
-        // These mark the user arguments.  We remove the first and last as they are the markers.
+    // See if we can find the `cargo-miri` markers. Those only get added to the binary we want to
+    // run. They also serve to mark the user-defined arguments, which we have to move all the way
+    // to the end (they get added somewhere in the middle).
+    let needs_miri = if let Some(begin) = args.iter().position(
+            |arg| arg == "cargo-miri-marker-begin") {
+        let end = args.iter().position(|arg| arg == "cargo-miri-marker-end")
+                      .expect("Cannot find end marker");
```

Comment:
why was this line changed?

Reply:
Just line-wrapped I think?

Reply:
revert or go full block indent then.

Reply:
@solson The issue is, the Rustc codebase uses this everywhere. :-P

Reply:
But I'll go to block comments...

Reply:
@alexreg Yeah, the fmt-rfcs process discussed alignment vs. block indent quite a bit, in part because rustc used alignment a lot, but eventually decided on block indent. If rustc starts using rustfmt with the default config it will start using block indent everywhere, as I understand it.